UPGRADE-1.15.md

```diff
@@ -0,0 +1,65 @@
+# UPGRADE FROM 1.x to 1.15
+
+## Method signature changes
+
+### Parameter types
+
+Starting with 1.14, methods now declare types for their arguments. This will not
+cause BC breaks unless you've passed a type that was incompatible with the type
+documented in the PHPDoc comment. A list of changes can be found at the bottom
```

Comment:
```suggestion
previously documented in the PHPDoc comment. A list of changes can be found at the bottom
```

Feel free to ignore if this goes without saying, but I think this was the the intended message.